packages/components/Button/__tests__/win32/ButtonLegacy.test.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
```

Comment:
It looks like the tests are duplicated in iOS and win32 - I wonder if it's possible to move the contents of the file (not the file itself) into another file ex. we'd have a ButtonLegacy.test.tsx file outside of iOS and win32 that contains any tests that are shared amongst all the platforms, and it exports those tests, and then there's a ButtonLegacy.test.tsx file in each of iOS/win32 that's referencing those tests. 

Not sure if this would be more complicated to navigate vs. potentially having to add a test in 5 different places

Reply:
Hmm yeah good idea. I'll try it out.

Reply:
I think the lack of code duplication is a win over having a hard time navigating to the test. Better that all our tests are consistent, imo.